dist/APIManager.js

```diff
@@ -0,0 +1,77 @@
+//This is the class that will manage all your APIs
+
+class APIManager {
+
+    constructor(renderer)
+     {
+         this.cityData = []
+         this.renderer = renderer
+    }
+    getDataFromDB()
```

Comment:
try to name your methods something more descriptive. Like `getCities()` and you do not need to say FromDB. People reading your code will know this